GVRf/Framework/src/org/gearvrf/scene_objects/GVRCubeSceneObject.java

```diff
@@ -71,16 +87,44 @@
             SIZE, -SIZE, SIZE, // 23
     };
 
```

Comment:
Please follow camel naming in Java. It means, simple_vertices should be like simpleVertices. And also, optional you can follow Android coding standards. For member variable, use m prefix, here it will be mSimpleVertices. For global reference use s-prefix, here it will be sSimpleVertices.

For these hard-coded parameters, I think they can be static. It means you don't have to keep one copy for each GVRCubeSceneObject.